core/templates/services/UpgradedServices.ts

```diff
@@ -963,6 +965,9 @@ export class UpgradedServices {
     upgradedServices['AutogeneratedAudioPlayerService'] =
       new AutogeneratedAudioPlayerService(
         upgradedServices['SpeechSynthesisChunkerService']);
+    upgradedServices['BottomNavbarStatusService'] = (
```

Comment:
This needs to be added to oppia-angular-root component. I guess you are now familiar with the process. However, just in case, Point 12 of https://github.com/oppia/oppia/wiki/Angular-Migration#implementation-details-to-migrate-services.

Reply:
Since this change has been made in the other pr, this is no longer required. But in that case, you should merge that other pr first.

Reply:
Hey @srijanreddy98 I think it's not required in this PR, I did the exact change [here](https://github.com/oppia/oppia/pull/9719#discussion_r446941406). So once that's merged, this issue will be resolved.